### PR TITLE
Add Playback.app version 1.1.0

### PR DESCRIPTION
**Pull request by vinkla**
Video player build using atom-shell and node.js https://github.com/mafintosh/playback

### CODE REVIEW

File: Casks/playback.rb [hunk 1]
```diff
@@ -0,0 +1,11 @@
+cask :v1 => 'playback' do
+  version '1.1.0'
+  sha256 'a9849a894e07e0b4f0c60095275752848117fb1d8e5de7781737bc03553fc708'
+
+  url "https://github.com/mafintosh/playback/releases/download/v#{version}/Playback.app.zip"
+  name 'Playback'
+  homepage 'https://github.com/mafintosh/playback'
```
Comment (vitorgalvao) [inline Casks/playback.rb]: https://mafintosh.github.io/playback/

File: Casks/playback.rb [hunk 2]
```diff
@@ -0,0 +1,11 @@
+cask :v1 => 'playback' do
+  version '1.1.0'
+  sha256 'a9849a894e07e0b4f0c60095275752848117fb1d8e5de7781737bc03553fc708'
+
+  url "https://github.com/mafintosh/playback/releases/download/v#{version}/Playback.app.zip"
```
Comment (vitorgalvao) [inline Casks/playback.rb]: Needs a `# github.com is the official download host per the vendor homepage` after the homepage change is made.

Also, why not [add an appcast](https://github.com/caskroom/homebrew-cask/pull/10056/files).

File: Casks/playback.rb [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+  name 'Playback'
+  homepage 'https://github.com/mafintosh/playback'
+  license :oss
```
Comment (vitorgalvao) [inline Casks/playback.rb]: :mit.

**Comment by vinkla** [comment]
All done. Also pushed a new version `1.2.0`.

**Comment by vitorgalvao** [comment]
Thank you.

**Pull request merged**